### PR TITLE
test: Fixing flaky specs

### PR DESCRIPTION
**Pull request by sarojsarab**
## Description
- Fixed flakyness in ForkApplication_spec.ts
- Removed FirstTimeUserOnboarding_spec.js as the the tests/scenarios covered are not relevant anymore due to new changes and the required changes has been covered in StartFromData_spec.ts and StartFromScratch_spec.ts already

## Summary by CodeRabbit

- **Tests**
    - Enhanced test automation by integrating `agHelper` and `assertHelper` functions, improving efficiency and reliability in test execution.

### CODE REVIEW

**Review by Aishwarya-U-R — APPROVED** [review]

File: app/client/cypress/e2e/Regression/ClientSide/Fork/ForkApplication_spec.ts [hunk 1]
```diff
@@ -130,7 +125,7 @@ describe(
     it("3. Mark application as forkable", () => {
       homePage.LogintoApp(Cypress.env("USERNAME"), Cypress.env("PASSWORD"));
       featureFlagIntercept({ license_gac_enabled: true });
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Fork/ForkApplication_spec.ts:127]: Can we use EnableGAC here? 
If the spec in unstable in EE runs, can we do this?

File: app/client/cypress/e2e/Regression/ClientSide/Fork/ForkApplication_spec.ts [hunk 2]
```diff
@@ -39,22 +42,23 @@ describe(
...
-      cy.wait(4000);
-      cy.wait("@postForkAppWorkspace")
-        .its("response.body.responseMeta.status")
-        .should("eq", 200);
-      cy.wait("@getWorkspace");
-      // check that forked application has same dsl
+      agHelper.WaitUntilEleAppear(homepagelocators.searchInput);
+      agHelper.GetElement(homepagelocators.searchInput).type(appname);
+      agHelper.WaitUntilEleAppear(homepagelocators.appMoreIcon);
+      agHelper
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Fork/ForkApplication_spec.ts:50]: Also can we use GetNClick here?

File: app/client/cypress/e2e/Regression/ClientSide/Fork/ForkApplication_spec.ts (hunk 1 above)
Comment (sarojsarab) [reply app/client/cypress/e2e/Regression/ClientSide/Fork/ForkApplication_spec.ts]: Done! Thanks for the suggestion.

File: app/client/cypress/e2e/Regression/ClientSide/Fork/ForkApplication_spec.ts (hunk 2 above)
Comment (sarojsarab) [reply app/client/cypress/e2e/Regression/ClientSide/Fork/ForkApplication_spec.ts]: Done.

**Review by Aishwarya-U-R — APPROVED** [review]

**Pull request merged**